src/index.ts

```diff
@@ -0,0 +1,102 @@
+import {CommitHeader, parseHeader} from "@kevits/conventional-commit";
+import {GitHub} from "@actions/github/lib/utils";
+
+const github = require('@actions/github');
+const core = require('@actions/core');
+
+let cachedOctokit: InstanceType<typeof GitHub>;
+
+const run = async () => {
+    const pullRequestTitle = github.context?.payload?.pull_request?.title;
```

Comment:
Be sure to document what events this action can be triggered by. I assume `pull_request` and `pull_request_target` will provide the same PR title, but if it's triggered by `push` then we'll have to figure out what PRs are connected to that branch. PR and PR target would be the safest in v1 I think.